client/http.go

```diff
@@ -216,6 +219,22 @@ func (c *HTTPClient) CommitInfo(ctx context.Context, stepID, branch string) (typ
 	return resp, err
 }
 
+// Healthz pings the healthz endpoint
+func (c *HTTPClient) Healthz(ctx context.Context) error {
+	req, err := http.NewRequestWithContext(ctx, "GET", healthzEndpoint, nil)
+	if err != nil {
+		return err
+	}
+	response, err := c.client().Do(req)
```

Comment:
there is a `do` function already, just reuse that

Reply:
done